algorithms/bfs/Bfs.py

```diff
@@ -0,0 +1,48 @@
+from collections import defaultdict
+
+class Graph:
```

Comment:
let's move this to graph folder.
I think we can merge this with [this](https://github.com/keon/algorithms/blob/master/algorithms/graph/graph.py#L89)